scripts/apps/authoring/metadata/views/metadata-dropdown.html

```diff
@@ -1,12 +1,15 @@
 <div class="dropdown dropdown--noarrow dropdown--dropleft" dropdown sd-metadropdown-focus>
+    <!-- if field is other than 'place', 'genre', 'anpa_category', 'subject', 'authors'-->
```

Comment:
It's better not to list the exact fields, because they might change and get out of sync with this comment. It would be more helpful if you commented on what is exceptional about this group of fields. If I understand correctly from the name `listFields` it probably means that one button is for multi-value inputs and another for single value inputs. It would be useful to rename the variable to `multiInputFields` if that's the case.